packages/dx-scheduler-core/src/utils.ts

```diff
@@ -307,6 +307,8 @@ export const calculateRectByDateIntervals: CalculateRectByDateIntervalsFn = (
 const expandRecurrenceAppointment = (
   appointment: AppointmentMoment, leftBound: Date, rightBound: Date,
 ) => {
+  const rightBoundUTC = new Date(getUTCDate(rightBound));
+  const leftBoundUTC = new Date(getUTCDate(leftBound) - 1);
```

Comment:
Why we are using `-1` here?

Reply:
Can we use `inc=true` instead of `-1`? See doc [here](https://github.com/jakubroztocil/rrule#rruleprototypebetweenafter-before-incfalse--iterator).